lib/rubocop/cop/chef/deprecation/legacy_notify_syntax.rb

```diff
@@ -78,7 +78,7 @@ def on_send(node)
                                 else
                                   "\"#{type.source}[\#{#{name.source}}]\""
                                 end
-                new_val = "#{notify_type} #{action.source}, #{service_value}".dup
+                new_val = +"#{notify_type} #{action.source}, #{service_value}"
```

Comment:
Is this string frozen? I wouldn't expect a dynamically built string to be automatically frozen somehow..

Reply:
Huh, I just tried it out and it is. TIL.